src/test/java/com/github/wasiqb/coteafs/datasource/JsonTest.java

```diff
@@ -0,0 +1,73 @@
+package com.github.wasiqb.coteafs.datasource;
+
+import static com.google.common.truth.Truth.assertWithMessage;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+import com.github.wasiqb.coteafs.datasource.data.JsonTestData;
+import com.github.wasiqb.coteafs.datasource.data.XmasFifthDay;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @author Faisal Khatri
+ * @since Aug 29, 2020
+ */
+public class JsonTest {
+
+    /**
+     * @author Faisal Khatri
+     * @since Aug 31, 2020
+     * @return testdata
+     */
+    @Test
```

Comment:
How can this be a test?

Reply:
Remove @Test annotation.

Reply:
Mark the comment as resolved once your changes are done.